templates/CRM/Group/Form/Delete.tpl

```diff
@@ -14,7 +14,7 @@
     <div class="messages status no-popup">
         <img src="{$config->resourceBase}i/Inform.gif" alt="{ts}status{/ts}"/>
     {ts 1=$title}Are you sure you want to delete the group %1?{/ts}<br /><br />
-    {if $count}
+    {if $count !== NULL}
         {ts count=$count plural='This group currently has %count members in it.'}This group currently has one member in it.{/ts}
```

Comment:
Just a comment about  terminology in CiviCRM. I think it creates confusion when we refer to group contacts as "members". "Groups" do not contain "members", they contain "contacts". The only entity in CiviCRM that contains "members" is a "Membership". But I digress...
This PR is an improvement and happy to merge it whether you feel like updating this text or not.

Reply:
Good point. Worth the new translations though?

Reply:
Good question. Maybe worth it, maybe not, we can merge this while thinking it over...